### PR TITLE
Intercept API: Command Log

### PR DESCRIPTION
**Pull request by mccataldo**
Closes DX-224

### CODE REVIEW

**Review by robertguss — APPROVED** [review]

**Comment by mccataldo** [comment]
On hold until #14843 lands

**Comment by flotwig** [comment]
Going to open a new PR for this now that https://github.com/cypress-io/cypress/pull/16730 is landed.

**Pull request closed without merging**
